src/ros_qos_specifier.cpp

```diff
@@ -0,0 +1,34 @@
+#include "realsense_gazebo_plugin/ros_qos_specifier.hpp"
+
+image_transport::CameraPublisher ImageTransportWithSpecifiedQos::advertise_camera(
+        const std::string & base_topic, uint32_t queue_size, rmw_qos_profile_t qos_profile) {
+    rmw_qos_profile_t custom_qos = qos_profile;
```

Comment:
no need for intermediate copy, you already have done copying in the argument list

Reply:
fixed